packages/core/src/shared/lsp/manifestResolver.ts

```diff
@@ -71,12 +80,12 @@ export class ManifestResolver {
     }
 
     private async getLocalManifest(): Promise<Manifest> {
-        logger.info('Failed to download latest LSP manifest. Falling back to local manifest.')
+        logger.info(`Failed to download latest ${this.lsName} manifest. Falling back to local manifest.`)
```

Comment:
same for the other cases. these names can have whitespace, right? Developers/users shouldn't have to manually parse the variable parts of a log message (which are often the important parts).

```suggestion
        logger.info(`Failed to download latest "${this.lsName}" manifest. Falling back to local manifest.`)
```